_dashboards/visualize/maptiles.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: Configuring a Web Map Service (WMS)
+grand_parent: Building data visualizations
+parent: Using coordinate and region maps
+nav_order: 5
+redirect_from:
+  - /docs/opensearch-dashboards/maptiles/
+---
+
+{%- comment -%}The `/docs/opensearch-dashboards/maptiles/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
+
+# Configuring a Web Map Service (WMS)
+
+The Open Geospatial Consortium (OGC) Web Map Service (WMS) specification is an international specification for requesting dynamic maps on the web. OpenSearch Dashboards includes default map tiles. For specialized maps, you can configure a WMS on OpenSearch Dashboards following these steps:
+
+1. Open and log in to OpenSearch Dashboards at `https://<host>:<port>`. Alternatively, you can connect to OpenSearch Dashboards by connecting to [https://localhost:5601](https://localhost:5601). The default username and password are `admin`. 
```

Comment:
nit - maybe just say

"Log into the OpenSearch Dashboards. . .

Reply:
Revised

Reply:
Revised